src/utils/transaction.ts

```diff
@@ -5,11 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-
 import { Utxo } from '../wallet/types';
 import { UtxoError } from '../errors';
+import {crypto as cryptoBL, PrivateKey} from 'bitcore-lib'
 
 const transaction = {
+  /**
+   * Get the signature from the dataToSignHash for a private key
+   *
+   * @param {Buffer} dataToSignHash sha256 of a transaction's dataToSign.
```

Comment:
This is not only the sha256 of data to sign, we need to do a little bit more. You can say that it's the hash of `dataToSign`